packages/services/hmi-server/build.gradle

```diff
@@ -68,7 +67,7 @@ dependencies {
 }
 
 group "software.uncharted"
-version "1.0.0-SNAPSHOT"
+version "2.0.0"
```

Comment:
Should this stay -SNAPSHOT since this is the development branch?

Reply:
I don't really now, `main` branch is the actual version we are deploying.